mergify_engine/context.py

```diff
@@ -214,11 +215,23 @@ def get_consolidated_data(self, name):
         else:
             raise AttributeError(f"Invalid attribute: {name}")
 
+    def update_check_runs(self, check):
+        self.check_runs = [c for c in self.check_runs if c["name"] != check["name"]]
+        self.check_runs.append(check)
+
+    @functools_bp.cached_property
+    def check_runs(self):
```

Comment:
```suggestion
    def pull_check_runs(self):
```

I think we were going for the `pull` prefix for things related to the PR?